NLibrary/Classes/DashboardHomeMVVM/DashboardViewModel.swift

```diff
@@ -2,9 +2,9 @@
 import Foundation
 import AVKit
 
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Limit vertical whitespace to a single empty line. Currently 2.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5382838)